src/catalog/src/durable/upgrade.rs

```diff
@@ -40,11 +40,14 @@
 //!    `objects_v15.proto` and `objects_v16.proto`. In this migration code you __should not__
 //!    import any defaults or constants from elsewhere in the codebase, because then a future
 //!    change could then impact a previous migration.
-//! 9. Call your upgrade function in [`run_upgrade()`].
-//! 10. Generate a test file for the new version:
+//! 9. Add an import for your new module to this file: mod v<CATALOG_VERSION-1>_to_v<CATALOG_VERSION>;
+//! 10. Call your upgrade function in [`run_upgrade()`].
+//! 11. Generate a test file for the new version:
 //!     ```ignore
 //!     cargo test --package mz-catalog --lib durable::upgrade::tests::generate_missing_encodings -- --ignored
 //!     ```
+//!     You'll need to update the hashes in `src/catalog/protos/hashes.json` based on the results
+//!     of that command, and add a new one for the new version.
```

Comment:
This is actually not specific to this command but a result of any time you try and build materialize after changing one of the proto files. Can you add this as a separate step after step 5?